CHANGELOG.md

```diff
@@ -9,3 +9,4 @@ See [UPGRADE.md](UPGRADE.md) for notes on breaking changes for downstream develo
 - Ease of use update to signatures, https://github.com/mafintosh/hypercore/issues/260
 - Updates [noise-protocol](https://github.com/emilbayes/noise-protocol) to latest, which uses chacha instead of xchacha and moves avoid from the sodium kx api for better compatability with the rest of the Noise ecosystem.
 - Updates [sodium-native](https://github.com/sodium-friends/sodium-native) from 2 to 3 across the board. 3 uses n-api, so no more builds needed when Node is released.
+- Update the discovery key to hash in the protocol version for better migration
```

Comment:
this is from the first commit which i force pushed out. can you remove that commit as well?

Reply:
droped it